.github/workflows/build-and-test.yml

```diff
@@ -87,6 +87,12 @@ jobs:
         name: test-log-${{ matrix.build.runs-on }}
         path: pytest.log
 
+    - name: Run Perf Benchmark
```

Comment:
Should we run these on a nighly basis, rather than every commit?

Reply:
Answered above. We're working on that. We still don't have any big model. When we add first bigger model like we will change that. Also, mnist is good to track all changes within the compiler, it's not big, and we can catch irregularities much faster.